bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -205,11 +209,16 @@ <h2>Webhook configuration</h2>
     </div>
     <ng-template #sendFinishedOverlay>
       <p>
-        If you subscribe your webhook to a task event of type triggered, the corresponding started and finished events
-        are sent automatically.
+        Sending started and finished from webhook service is an "async" model. The Keptn task immediately returns. Async
```

Comment:
I would shorten the paragraph a little bit by focusing on the benefit that the receiver does not need to send the finished event. 

My proposal: 
If you subscribe your webhook to a task event of type triggered, the corresponding finished event is sent automatically. Consequently, the receiving service does not need to send back the finished event.